profile/src/main/java/rdx/works/profile/domain/GetProfileUseCase.kt

```diff
@@ -136,3 +150,11 @@ val GetProfileUseCase.p2pLinks
  */
 suspend fun GetProfileUseCase.defaultDepositGuarantee() =
     invoke().map { it.appPreferences.transaction.defaultDepositGuarantee }.first()
+
+private fun List<Network.Account>.notHiddenAccounts(): List<Network.Account> {
```

Comment:
`notHidden` to avoid the repetition. ♻️ 
same for personas

Reply:
I get JVM platform declaration clash if I change naming, `Platform declaration clash: The following declarations have the same JVM signature (access$notHidden(Ljava/util/List;)Ljava/util/List;):`

Reply:
oh yes right its an extenstion fun. My bad.